src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -104,5 +108,29 @@ void shouldThrowAnExceptionWhenNameContainsIllegalLetters() throws Exception {
         result.andExpect(status().isBadRequest());
     }
 
+    @Test
+    void shouldFindWalletById() throws Exception {
+        // given
+        Instant creationDate = Instant.now();
+        WalletDTO wallet = new WalletDTO(1L, "", creationDate);
+        // when
+        when(walletService.findById(1L)).thenReturn(wallet);
+        ResultActions result = mockMvc.perform(get("/api/wallet/wallet?id=1")
+                .contentType(MediaType.APPLICATION_JSON));
```

Comment:
Please add one empty line after this "//when" block

Reply:
Please change also the path according to the https://github.com/bycza-zagroda/track-expenses-app-backend/pull/62#discussion_r1001001449